### PR TITLE
Fix StartTime styling bugs

### LINKED ISSUE #3301: Radio Schedules start time styling bugs
**Describe the bug**

- On page re-size on Safari (and on mobile breakpoint on Safari) the clock icon misses one pixel on the side and bottom
Screenshot 2020-03-25 at 13 39 05

- The line next to the start time doesn't appear on IE 11 on Windows 7, whereas it does on Firefox. We should check on browserstack.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to www.test.bbc.com/hausa?renderer_env=live
2. Scroll down to Radio schedule component (below top stories)
3. See error

**Expected behaviour**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #3301 

**Overall change:** _Fix clock icon being cut off on smaller breakpoints

_Added overflow styling to fix pixels being cut off._

**Before**
Screenshot 2020-03-31 at 17 52 25

**After**
Screenshot 2020-03-31 at 17 53 06

**Code changes:**
- _Add `overflow: visible` styling to clock icon via `StyledClock`._
- _Update snapshots._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -142,6 +142,10 @@ exports[`StartTime should render LTR correctly 1`] = `
     >
       14:54
     </time>
+    <span
```
Comment (greenc05) [inline packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap]: Not sure about adding an empty span just for a line, can't we live without this in IE??

Comment (tochwill) [reply packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap]: I'd rather we kept using the pseudo-element, and just didn't show the line in IE. We can run it past UX?

Comment (Bopchy) [reply packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap]: Will reach out to UX and revert

Comment (Bopchy) [reply packages/components/psammead-radio-schedule/src/StartTime/__snapshots__/index.test.jsx.snap]: ~~Fixing this issue on IE has been deemed as low priority for now, and as something that should be added to the tech debt backlog. I will revert the changes made to remove the pseudo styling and only include the fix for Safari in this PR.  ~~

~~**Note: Adding additional mark up to fix this is not an appropriate solution as mentioned by** @greenc05 [above](https://github.com/bbc/psammead/pull/3318#discussion_r401015655)~~

Reached out to UX. We're ok with not fixing this IE issue. I will revert changes and go back to using the pseudo-element styling.

**Review by OlgaLyubin — APPROVED** [review]
Thank you for the fix 👍Clock is no longer being cut off

**Review by EinsteinNjoroge — APPROVED** [review]
👍

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**